extensions/wikia/ArticleVideoMVP/scripts/ooyalaVideo.js

```diff
@@ -0,0 +1,43 @@
+define('ooyalaVideo', ['jquery', 'wikia.window', 'wikia.loader'], function ($, window, loader) {
+
+	function OoyalaVideo(elementId, ooyalaJsFile, ooyalaVideoId, ooyalaPlayerOptions) {
+		var self = this;
```

Comment:
both are ok - https://github.com/Wikia/guidelines/blob/master/JavaScript/CodingConventions.md#storing-context-in-a-local-variable